sdk/tables/azure-data-tables/samples/async_samples/sample_copy_table_async.py

```diff
@@ -44,10 +45,13 @@ def __init__(self):
             self.account_name, self.access_key, self.endpoint_suffix
         )
         self.table_name="mytable"
-        self.table_service_client = TableServiceClient.from_connection_string(conn_str=self.table_connection_string, table_name=self.table_name)
-        self.blob_connection_string = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
+        self.blob_account_name = os.getenv("STORAGE_ACCOUNT_NAME")
+        self.blob_account_key = os.getenv("STORAGE_ACCOUNT_KEY")
+        self.blob_connection_string = "DefDefaultEndpointsProtocol=https;AccountName={};AccountKey={};EndpointSuffix=core.windows.net".format(
+            self.blob_account_name, self.blob_account_key
+        )
         self.blob_service_client = BlobServiceClient.from_connection_string(self.blob_connection_string)
-        self.container_name = self.table_name
+        self.container_name = "mycontainer"
```

Comment:
Why do we need a separate container_name? Can we just use table_name throughout?

Reply:
We can't use the table_name throughout because we tear_down tables in each of the test, the `create table` will fail in the second sample test as the table with the same name haven't finish `delete`.